Mathlib/Analysis/Normed/Group/Basic.lean

```diff
@@ -988,6 +988,11 @@ theorem norm_toNNReal' : ‖a‖.toNNReal = ‖a‖₊ :=
 #align norm_to_nnreal' norm_toNNReal'
 #align norm_to_nnreal norm_toNNReal
 
+/-- The non negative norm seen as an `ENNReal` and then as a `Real` is equal to the norm. -/
+@[to_additive ofNNReal_toReal_nnnorm "The non negative norm seen as an `ENNReal` and
+then as a `Real` is equal to the norm."]
+theorem ofNNReal_toReal_nnnorm' : (‖a‖₊ : ℝ≥0∞).toReal = ‖a‖ := rfl
```

Comment:
`ofNNReal` is not a name we use for that coercion: let's use the generic `coe` instead. Also in names we describe a composition from the outside towards the inside, which is why I think the following name is better:
```suggestion
/-- The non negative norm seen as an `ENNReal` and then as a `Real` is equal to the norm. -/
@[to_additive toReal_coe_nnnorm "The non negative norm seen as an `ENNReal` and
then as a `Real` is equal to the norm."]
theorem toReal_coe_nnnorm' : (‖a‖₊ : ℝ≥0∞).toReal = ‖a‖ := rfl
```

Also, could you place that lemma next to another lemma relating nnnorm, norm and ENNReal: `ofReal_norm_eq_coe_nnnorm`?